contracts/BSCValidatorSet.sol

```diff
@@ -397,7 +397,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber {
     }else if (Memory.compareStrings(key, "expireTimeSecondGap")){
       require(value.length == 32, "length of expireTimeSecondGap mismatch");
       uint256 newExpireTimeSecondGap = BytesToTypes.bytesToUint256(32, value);
-      require(newExpireTimeSecondGap >=100 && newExpireTimeSecondGap <= 1e5, "the extraFee out of range");
+      require(newExpireTimeSecondGap >=100 && newExpireTimeSecondGap <= 1e5, "the expireTimeSecondGap out of range");
```

Comment:
should we update it to `the expireTimeSecondGap is out of range`

Reply:
sure